adapters/beachfront/beachfront.go

```diff
@@ -674,7 +685,7 @@ func getIP(ip string) string {
 	// and seems not to know what IPv6 is, so just setting it to one that is not likely to
 	// be used.
 	if ip == "" || ip == "::1" || ip == "127.0.0.1" {
```

Comment:
FYI: PBS-Core now filters out local IPv4 and IPv6 subnets. The `ip == "::1" ` and `ip == "127.0.0.1"` should now be unreachable.

Reply:
Thanks. I'll drop that.

Reply:
To clarify, it's still technically possible for `ip == ""` if the server can't find a public ip address on the request. Might want to still include the fallback if your bidding server absolutely requires an ip address.

Reply:
I'm reviewing how ip is handled. Please hold off until I commit those changes, which should be by Monday afternoon.

Reply:
Ok LGTM